src/hypothesis/searchstrategy/strategies.py

```diff
@@ -271,6 +279,14 @@ def do_draw(self, data):
                     raise
         reject()
 
+    @property
+    def branches(self):
+        branches = [
```

Comment:
Same question as for flatmap.

Also, map and flatmap seem to have got branches but filter hasn't. Is there a reason for that or is it an accidental omission?